insuree/schema.py

```diff
@@ -162,7 +164,13 @@ def resolve_insurees(self, info, **kwargs):
         filters = []
         additional_filter = kwargs.get('additional_filters', None)
         chf_id = kwargs.get('chf_id')
+        default_insuree_number_length = 50
+        chf_id_max_length = getattr(InsureeConfig, 'insuree_number_length', None) or default_insuree_number_length
```

Comment:
https://docs.python.org/3.8/library/functions.html#getattr
```suggestion
        chf_id_max_length = getattr(InsureeConfig, 'insuree_number_length', 50) or default_insuree_number_length
```